connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnectorTest.java

```diff
@@ -151,7 +155,14 @@ public void testFindConsumerGroups() throws Exception {
         doReturn(groups).when(connector).listConsumerGroups();
         doReturn(true).when(connector).shouldReplicateByTopicFilter(anyString());
         doReturn(true).when(connector).shouldReplicateByGroupFilter(anyString());
-        doReturn(offsets).when(connector).listConsumerGroupOffsets(anyString());
```

Comment:
I think this leaves the `offsets` variable unused.

Reply:
I have not used offsets anymore, but offsetsResult need to be mocked